### PR TITLE
Add non-coverage environment

### PR DESCRIPTION
**Pull request by wsanchez**
Rename `trial-*` environments to `coverage-*`.
Add `trial-*` environments that don't run under coverage.

---
This change is [Reviewable](https://reviewable.io/reviews/twisted/klein/189)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -47,11 +47,14 @@ deps =
 
 commands =
     "{toxinidir}/.travis/environment"
-    coverage run -p "{envdir}/bin/trial" {posargs:klein}
+
+    trial: "{envdir}/bin/trial" {posargs:klein}
```
Comment (twm) [inline tox.ini]: There is an `{envbindir}` variable this scenario. No need to change this, though.

Comment (glyph) [inline tox.ini]: Isn't `{envbindir}` `Scripts` on Windows, and hence, this does need changing?

Comment (glyph) [inline tox.ini]: Oops no it doesn't it was wrong before :) thanks for pointing it out though.

Comment (wsanchez) [inline tox.ini]: Fixed, thanks.  Forgot that variable was there.

**Pull request merged**